src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -245,14 +245,7 @@ private TraceWriter CreateDefaultTraceWriter(WebHostSettings settings)
             var systemEventGenerator = config.HostConfig.GetService<IEventGenerator>() ?? new EventGenerator();
             TraceWriter systemTraceWriter = new SystemTraceWriter(systemEventGenerator, _settingsManager, TraceLevel.Verbose);
 
-            // Note that we're creating a logger here that is independent of log configuration settings
```

Comment:
nit: the comment above is also wrong now "logs both host logs and system logs"

Reply:
Good catch! Updating comments.